Assets/SO Architecture/Editor/Inspectors/BaseVariableEditor.cs

```diff
@@ -42,15 +42,34 @@ public override void OnInspectorGUI()
             serializedObject.Update();
 
             DrawValue();
-
-            EditorGUILayout.Space();
-
             DrawClampedFields();
             DrawReadonlyField();
         }
         protected virtual void DrawValue()
         {
-            GenericPropertyDrawer.DrawPropertyDrawerLayout(_valueProperty, Target.Type);
+            using (var scope = new EditorGUI.ChangeCheckScope())
+            {
+                string content = "Cannot display value. No PropertyDrawer for (" + Target.Type + ") [" + Target + "]";

```

Comment:
If content isn't used it's going to be discontented.

Reply:
Lol, this is from the original library's source code, but if truly unused I have no qualms about removing it.

Reply:
Removed this unused code.